.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "test_helpers/bats-support"]
```

Comment:
We had a lot of bad experience with submodules, perhaps we should just use substrees to avoid potential issues?

Reply:
I don't see this getting any more complex than it is right now. I don't expect to do any development on the BATS assert helpers themselves, and they are going to go to great lengths to maintain reverse compatibility.

I think submodules will work here. What types of issues do you think using subtrees will avoid?